src/ophyd_async/epics/areadetector/drivers/vimba_driver.py

```diff
@@ -0,0 +1,58 @@
+from enum import Enum
+
+from ..utils import ad_rw
+from .ad_base import ADBase
+
+
+class VimbaPixelFormat(str, Enum):
+    internal = "Mono8"
+    ext_enable = "Mono12"
+    ext_trigger = "Ext. Trigger"
+    mult_trigger = "Mult. Trigger"
+    alignment = "Alignment"
+
+
+class VimbaConvertFormat(str, Enum):
+    none = "None"
+    mono8 = "Mono8"
+    mono16 = "Mono16"
+    rgb8 = "RGB8"
+    rgb16 = "RGB16"
+
+
+class VimbaTriggerSource(str, Enum):
+    freerun = "Freerun"
+    line1 = "Line1"
+    line2 = "Line2"
+    fixed_rate = "FixedRate"
+    software = "Software"
+    action0 = "Action0"
+    action1 = "Action1"
```

Comment:
The list is exhaustive, and I believe applicable to all Vimba cameras. Since this parameter comes from the Vimba API rather than the GenICam generated xml from Aravis, we can assume these are universally the same.